src/ClientHandle.h

```diff
@@ -453,7 +453,12 @@ class cClientHandle  // tolua_export
 		// TODO: Add Kicking here as well
 	} ;
 	
+#ifdef ANDROID
+#pragma message "Android <atomic> support is incomplete or broken - using plain variable (subject to data races!)"
+	eState m_State;
```

Comment:
I don't think you should be bothered about this, considering it may well be fixed anyway in future NDK releases and Android runs at *0.8 chunks per second* and so is unlikely to be used as an actual server. I definitely don't have the time to try and re-implement atomic :P